integration_tests/toMatchSnapshot/__tests__/basic-support.test.js

```diff
@@ -0,0 +1 @@
+test('snapshots', () => expect({apple: "updated value"}).toMatchSnapshot());
```

Comment:
I think my tests went badly. I was running on Windows and a bunch of stuff was failing. Will fix now that I'm back on the mac.